data-archive/archive-jobs/mosip_archive_main.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import sys
+import psycopg2
+import configparser
+import datetime
+import os
+import json
+from datetime import datetime
+
+def config():
+    config = configparser.ConfigParser()
+    if os.path.exists('db.properties'):
+        config.read('db.properties')
+        archive_param = {key.upper(): config['ARCHIVE'][key] for key in config['ARCHIVE']}
+        source_param = {db_name: {key.upper(): config[db_name][key] for key in config[db_name]} for db_name in config.sections() if db_name != 'ARCHIVE'}
+        
+        # Read database names from properties file
+        db_names = config.get('Databases', 'DB_NAMES').split(',')
+        db_names = [name.strip() for name in db_names]  # Strip leading and trailing spaces
+        print("db.properties file found and loaded.")
+    else:
+        print("db.properties file not found. Using environment variables.")
+        archive_param = {
+            'ARCHIVE_DB_HOST': os.environ.get('ARCHIVE_DB_HOST'),
+            'ARCHIVE_DB_PORT': os.environ.get('ARCHIVE_DB_PORT'),
+            'ARCHIVE_DB_NAME': os.environ.get('ARCHIVE_DB_NAME'),
+            'ARCHIVE_SCHEMA_NAME': os.environ.get('ARCHIVE_SCHEMA_NAME'),
+            'ARCHIVE_DB_UNAME': os.environ.get('ARCHIVE_DB_UNAME'),
+            'ARCHIVE_DB_PASS': os.environ.get('ARCHIVE_DB_PASS')
+        }
+        db_names_env = os.environ.get('DB_NAMES')
+        if db_names_env is not None:
+            db_names = db_names_env.split(',')
+            db_names = [name.strip() for name in db_names]  # Strip leading and trailing spaces
+        else:
+            print("Error: DB_NAMES not found in properties file or environment variables.")
+            sys.exit(1)
+
+        source_param = {}
+        for db_name in db_names:
+            source_param[db_name] = {
+                f'{db_name}_SOURCE_DB_HOST': os.environ.get(f'{db_name}_SOURCE_DB_HOST'),
+                f'{db_name}_SOURCE_DB_PORT': os.environ.get(f'{db_name}_SOURCE_DB_PORT'),
+                f'{db_name}_SOURCE_DB_NAME': os.environ.get(f'{db_name}_SOURCE_DB_NAME'),
+                f'{db_name}_SOURCE_SCHEMA_NAME': os.environ.get(f'{db_name}_SOURCE_SCHEMA_NAME'),
+                f'{db_name}_SOURCE_DB_UNAME': os.environ.get(f'{db_name}_SOURCE_DB_UNAME'),
+                f'{db_name}_SOURCE_DB_PASS': os.environ.get(f'{db_name}_SOURCE_DB_PASS')
+            }
```

Comment:
first, check for env vars if it does not exist then take values from db.properties